### DISCUSSION TITLE
Implement a Dropdown

### DISCUSSION THREAD

**Discussion by herijaya** [godruoyi/gblog#39] [Ideas]
Simple Dropdown A minimal dropdown that can expand when hovered or clicked, revealing more subcategories.

Would it be possible to consider adding this feature to gblog? I believe this would greatly benefit users, particularly those managing blog or websites with a complex structure. The implementation could be made configurable, so it can be enabled or customized as per the user’s needs.

Best regards,
MAXsi.ID

**Comment by godruoyi** [comment]
Do you mean the example below? I think we can do it ourselves since it's too simple to add this. You just need to copy this demo code into the gblog and it should work fine.

https://preline.co/examples/navigations-mega-menu.html#dropdown-style-with-icon-and-description
